packages/react-recommendations/README.md

```diff
@@ -326,4 +478,22 @@ Additional filters to use as fallback should there not be enough recommendations
 
 > [`SearchParameters`](https://www.algolia.com/doc/api-reference/search-api-parameters/) | defaults to `{ analytics: false, enableABTest: false }`
 
+### `transformItems`
+
+> `(Array<RecordWithObjectID<TItem>>) => Array<RecordWithObjectID<TItem>>`
+
+<details>
+
+<summary><code>RecordWithObjectID</code></summary>
+
+```ts
+type RecordWithObjectID<TItem> = TItem & {
+  objectID: string;
+};
+```
+
+</details>
+
+Receives the items, and is called before displaying them. Should return a new array with the same shape as the original array. Useful for mapping over the items to transform, and remove or reorder them.
```

Comment:
We can move this function above `details`.

```suggestion
Function to transform the items retrieved by Algolia. It's useful to edit, add, remove or reorder them.
```

Reply:
Done in https://github.com/algolia/ui-components/pull/18/commits/3335abd6b66e9621bb5985eb477a8cf869c2d82b